server/src/utils/mixerConnections/LawoRubyConnection.ts

```diff
@@ -251,36 +255,37 @@ export class LawoRubyMixerConnection {
             if (node.contents.type !== Model.ElementType.Parameter) return
 
             logger.debug(`Subscription of channel level: ${command}`)
-            this.emberConnection.subscribe(node, () => {
-                logger.trace(`Receiving Level from Ch ${ch}`)
-                if (
-                    !state.channels[0].chMixerConnection[this.mixerIndex]
-                        .channel[ch - 1].fadeActive &&
-                    (node.contents as Model.Parameter).value as number >
-                        this.mixerProtocol.channelTypes[typeIndex].fromMixer
-                            .CHANNEL_OUT_GAIN[0].min
-                ) {
-                    store.dispatch(
-                        storeFaderLevel(
-                            ch - 1,
-                            dbToFloat(
-                                (node.contents as Model.Parameter)
-                                    .value as number
-                            )
-                        )
-                    )
-                    global.mainThreadHandler.updatePartialStore(ch - 1)
-                    if (remoteConnections) {
-                        remoteConnections.updateRemoteFaderState(
-                            ch - 1,
-                            dbToFloat(
-                                (node.contents as Model.Parameter)
-                                    .value as number
+            this.emberConnection.subscribe(
+                node as NumberedTreeNode<EmberElement>,
+                () => {
+                    const levelDB = (node.contents as Model.Parameter)
+                        .value as number

```

Comment:
Is `DB` short for decibel? Given the context I assume, yes. It would be nice if the acronym was spelled out since a more common usage of the acronym for `db` is `database`.

I'm aware that you just extracted this to a local variable, but since you extracted it, you should use type declaration over type assertion i.e. `const someVariable: number = ...` over `const someVariable = ... as number`.

Reply:
Since `(node.contents as Model.Parameter).value` can be something else than a number it doesn't seem possible to use a declaration here.

Reply:
@LindvedKrvang I'm really concerned about the level of details the Sofie team is going into in their review.

If your argument should hold, db should never be used as acronym for Database.
Everyone with just a base knowledge of audiomixers and audioprotocols knows what levelDB means.
A review should be constructive and not counterproductive. If not, then the time is better spend actually running the sourcecode and see how it behaves. IMHO.